hack/install-router.sh

```diff
@@ -32,8 +32,23 @@ fi
 
 OPENSHIFT_INSECURE="${OPENSHIFT_INSECURE:-false}"
 OPENSHIFT_CA_DATA="${OPENSHIFT_CA_DATA:-}"
+OPENSHIFT_CERT_DATA="${OPENSHIFT_CERT_DATA:-}"
+OPENSHIFT_KEY_DATA="${OPENSHIFT_KEY_DATA:-}"
 
 if [[ "${MASTER_URL}" == "https"* ]]; then
+	# Read client cert data in to send to containerized components
+	OPENSHIFT_CA_DATA="$(cat "${CERT_DIR}/root.crt")"
+	OPENSHIFT_CERT_DATA="$(cat "${CERT_DIR}/cert.crt")"
+	OPENSHIFT_KEY_DATA="$(cat "${CERT_DIR}/key.key")"
+
+	# I don't know how to do this inline with bash and it's logically a separate step we want to remove anyway
+	# TODO: remove this once services can provide root cert data to pods
+	# Escape cert data for json
+	OPENSHIFT_CA_DATA="${OPENSHIFT_CA_DATA//$'\n'/\\\\n}"
+	OPENSHIFT_CERT_DATA="${OPENSHIFT_CERT_DATA//$'\n'/\\\\n}"
+	OPENSHIFT_KEY_DATA="${OPENSHIFT_KEY_DATA//$'\n'/\\\\n}"
+
+
 	if [[ "$OPENSHIFT_CA_DATA" == "" ]]; then
 		echo "Running against an HTTPS master (${MASTER_URL}) without a trusted certificate bundle."
 		echo "Set \$OPENSHIFT_CA_DATA to the contents of the root certificate bundle to start securely next time."
```

Comment:
Oops... update this to describe the correct invocation (`CERT_DIR=openshift.local.certificates/openshift-client ...`)